Library/Magic/Magic Spells.spl

```diff
@@ -19,6 +19,8 @@
 			"maintenance_cost": "Varies",
 			"casting_time": "2 sec",
 			"duration": "1 min",
+			"prereq_count": 20,
+			"prereq_count": 2,
```

Comment:
This looks broken.

Reply:
I did a few inside GCS, but it was too slow, so I did the rest in vim where I had macros, copying numbers from the tables in the back of Magic.  I loaded the file into GCS when I finished to make sure it was still well-formed JSON but clearly missed that double-entry above, sorry.  (I wouldn't be surprised if there were a couple more typos as this was hundreds of numbers.)

Reply:
The new "id" and "sv" stuff under weapons were added by GCS not me.

Reply:
Yeah, the 'sv' field is new-ish and expected. The changing IDs are not, but I haven't yet tracked down why that happens sometimes.